van-gogh-gallery/josefin/pages/about.css

```diff
@@ -0,0 +1,191 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap');
+
+* {
+    margin: 0px;
+    padding: 0px;
+    font-family: 'Montserrat', sans-serif;
+    
+}
+
+.nav-container{
+    position: absolute;
+   
+    margin-left: 0px;
+    margin-bottom: 10px;
+    cursor: pointer;
+    transition: all 0.2s;
+    font-size: 20px;
+font-weight: bolder;
```

Comment:
This won't work cause your import link contains only font-weight: 200 `@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap');`

Reply:
updated